src/web_socket.toit

```diff
@@ -232,14 +275,19 @@ class WebSocket:
   */
   close_write --status_code/int=STATUS_WEBSOCKET_NORMAL_CLOSURE:
     if current_writer_ == null:
+      writer_semaphore_.down
```

Comment:
Do you need a finally here, and a critical do?
if we get caught by a timeout exception I'm not sure we will correctly call the semaphore_.up otherwise.